facebook-commerce.php

```diff
@@ -1485,7 +1485,12 @@ function update_product_group( $woo_product ) {
 		);
 
 		// Figure out the matching default variation.
-		$default_product_fbid = $this->get_product_group_default_variation( $woo_product, $fb_product_group_id );
+		$default_product_fbid = apply_filters(
```

Comment:
I suggest applying the filter within the `get_product_default_variation` method rather:

https://github.com/woocommerce/facebook-for-woocommerce/blob/0b4eecc7de4e07bf02d84b746ff4bab0ee0c84aa/facebook-commerce.php#L1570


I would also recommend adding a doc block to the new filter as done here:

https://github.com/woocommerce/facebook-for-woocommerce/pull/2169/files#diff-8f3102bd79b0b163d0cbb9a7341231014b0dcd6a70d4cd25cdab2dc8f432ba9bR104-R112

Reply:
@rawdreeg - Thank you for the comments. I've pushed changes. Could you please take a look?